src/profile-logic/call-tree.js

```diff
@@ -601,8 +601,17 @@ export function getCallTree(
 }
 
 /**
- * This function takes the call tree summary strategy, and finds the appropriate data
- * structure. This can then be used by the call tree and other UI to report on the data.
+ * Returns a table with the appropriate data for the call tree summary strategy,
+ * for use by the call tree or flame graph.
+ *
+ * If the strategy is one of the allocation strategies, the returned table will be
+ * based on thread.jsAllocations or thread.nativeAllocations, but with a modified
+ * stack column: Some samples will have a null stack, so that they are ignored.
```

Comment:
note: my understanding is that your change applies only to nativeAllocations, not jsAllocations, so IMO this comment isn't accurate.

```suggestion
 * If the strategy is one of the native allocation strategies, the returned table
 * will be based on thread.nativeAllocations, but with a modified
 * stack column: Some samples will have a null stack, so that they are ignored.
```